theseus_gui/src/pages/Browse.vue

```diff
@@ -505,6 +506,19 @@ const showLoaders = computed(
       instanceContext.value === null) ||
     ignoreInstanceLoaders.value
 )
+
+const offline = ref(await isOffline())
+// Immediately return to home page if offline
+if (offline.value) {
+  await router.push('/')
+}
+const unlisten = await offline_listener((e) => {
+  if (e) {
+    router.push('/')
+  }
+})
+
+onUnmounted(() => unlisten())
```

Comment:
same here- don't think this is neccessary- it's fine if it infinitley loads if the user is offline